File: Java/Lv.0_홀짝에 따라 다른 값 반환하기.java
// Java

class Solution {
    public int solution(int n) {
        int answer = 0;
        
        if (n % 2 == 1) {
            for (int i=n; i>=1; i-=2) {
            answer += i;    
            }
        } else {
            for (int i=n; i>0; i-=2) {
                answer += Math.pow(i,2);
            }
        }
        
        return answer;
    }
}

File: JavaScript/Lv.0_홀짝에 따라 다른 값 반환하기.js
// JavaScript

function solution(n) {
    var answer = 0;
    if (n % 2 == 1) {
        for (let i=n; i>0; i-=2) {
            console.log(i);
            answer += i;
            console.log(answer);
        }
    } else {
        for (let i=n; i>0; i-=2) {
            console.log(i);
            answer += (i*i);
            console.log(answer);
        }
    }
    return answer;
}

// Math.pow(n,2)를 통해 제곱으로 만들 수 있음.